lib/msal-common/src/utils/Constants.ts

```diff
@@ -34,7 +34,6 @@ export const Constants = {
     CODE_GRANT_TYPE: "authorization_code",
```

Comment:
You may want to clean some of these as they have their own enums defined in the same file below!